### PR TITLE
MAE-8 Video widget url inline editor - implementation

### PR DESCRIPTION
**Pull request by LadislavMargai**

### CODE REVIEW

File: Kentico.Widget.Video/Kentico.Widget.Video.csproj [hunk 1]
```diff
@@ -536,13 +536,17 @@
...
   <ItemGroup>
-    <TypeScriptCompile Include="Assets\Widgets\Video\main.ts" />
+    <EmbeddedResource Include="App_Data\Global\Resources\Kentico.Widget.Video.resx">
+      <SubType>Designer</SubType>
+    </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup />
   <ItemGroup>
-    <EmbeddedResource Include="App_Data\Global\Resources\Kentico.Widget.Video.resx" />
+    <TypeScriptCompile Include="Assets\InlineEditors\Kentico.VideoUrlEditor\main.ts" />
```
Comment (petrinecp) [inline Kentico.Widget.Video/Kentico.Widget.Video.csproj]: Include this file in project.

File: types/kentico.ts [hunk 2]
```diff
@@ -2,11 +2,16 @@ type InlineEditorLifecycleCallback = (options: InlineEditorOptions) => void;
...
   readonly editor: HTMLElement;
-  readonly localization: any;
+  readonly localizationService: LocalizationService;
   readonly propertyValue: string;
   readonly propertyName: string;
 }
 
+interface LocalizationService {
+  readonly cultureCode: string;
+  getString: (resourceString: string, ...parameters: any[]) => string;
```
Comment (petrinecp) [inline types/kentico.ts]: Make this property readonly.

File: Kentico.Widget.Video/Kentico.Widget.Video.csproj (hunk 1 above)
Comment (petrinecp) [inline Kentico.Widget.Video/Kentico.Widget.Video.csproj]: Include this file in project.

Comment (LadislavMargai) [reply Kentico.Widget.Video/Kentico.Widget.Video.csproj]: Done

File: types/kentico.ts (hunk 2 above)
Comment (LadislavMargai) [reply types/kentico.ts]: Done.

File: Kentico.Widget.Video/Kentico.Widget.Video.csproj [hunk 3]
```diff
@@ -536,13 +536,17 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="app.config" />
+    <None Include="Assets\InlineEditors\Kentico.VideoUrlEditor\style.less" />
```
Comment (petrinecp) [inline Kentico.Widget.Video/Kentico.Widget.Video.csproj]: Please change the file build action to Content.

**Pull request merged**